src/redux/actions/inputAction.ts

```diff
@@ -0,0 +1,94 @@
+import { Dispatch } from "react";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import getData, {
+  changeTaskColumnIdRequest,
+  createColumnRequest,
+  createTaskRequest,
+  deleteColumnRequest,
+  deleteTaskRequest,
+} from "../../api";
+import { Column } from "../../components/Board";
+import { SET_APP_DATA } from "../constants";
+
+interface SetAppDataAction {
+  type: string;
+  payload: Column;
+}
+
+export const setAppDataAction = (data: Column): SetAppDataAction => ({
+  type: SET_APP_DATA,
+  payload: data,
+});
+
+async function handleRequestSuccess(dispatch: Dispatch<unknown>) {
+  try {
+    const data = await getData();
+    dispatch(setAppDataAction(data));
+  } catch (error) {
+    toast.error(error.message);
+  }
+}
+
+export const getAppDataAction = () => async (
+  dispatch: Dispatch<unknown>
```

Comment:
also here `Dispatch<unknown>` and further on.
we know what data's gonna be dispatched

Reply:
If I return another type different from Dispatch<unknown>, I get an error